test/integration/testHttpClient.js

```diff
@@ -0,0 +1,113 @@
+const connOptions = require('../integration/connectionOptions');
+const ConnectionConfig = require('../../lib/connection/connection_config');
+const NodeHttpClient = require('../../lib/http/node').NodeHttpClient;
+const { hangWebServerUrl } = require('../hangWebserver');
+const assert = require('assert');
+const testUtil = require('./testUtil');
+
+describe('HttpClient Tests', () => {
+  let httpClientInstance;
+
+  const connectionOptions = {
+    ...(connOptions.valid),
+    timeout: 100000
```

Comment:
why 10 seconds? not too much?

Reply:
You are right, changed to 2 seconds - should be enough.